superset-frontend/packages/superset-ui-chart-controls/package.json

```diff
@@ -2,41 +2,44 @@
   "name": "@superset-ui/chart-controls",
   "version": "0.18.25",
   "description": "Superset UI control-utils",
-  "sideEffects": false,
-  "main": "lib/index.js",
-  "module": "esm/index.js",
-  "files": [
-    "esm",
-    "lib"
-  ],
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/apache-superset/superset-ui.git"
-  },
   "keywords": [
     "superset"
   ],
-  "author": "Superset",
-  "license": "Apache-2.0",
+  "homepage": "https://github.com/apache-superset/superset-ui#readme",
   "bugs": {
     "url": "https://github.com/apache-superset/superset-ui/issues"
   },
-  "homepage": "https://github.com/apache-superset/superset-ui#readme",
-  "publishConfig": {
-    "access": "public"
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/apache-superset/superset-ui.git"
   },
+  "license": "Apache-2.0",
+  "author": "Superset",
+  "sideEffects": false,
+  "main": "lib/index.js",
+  "module": "esm/index.js",
+  "files": [
+    "esm",
+    "lib"
+  ],
   "dependencies": {
     "@react-icons/all-files": "^4.1.0",
     "lodash": "^4.17.15",
     "prop-types": "^15.7.2"
   },
   "peerDependencies": {
+    "@ant-design/icons": "^4.2.2",
     "@emotion/react": "^11.4.1",
     "@superset-ui/core": "*",
+    "@types/enzyme": "^3.10.5",
```

Comment:
> Codes LGTM! Thanks for bringing this Popover, and the SQL is more readable!
> 
> There are some no-blocking design suggestions:
> 
> 1. Could we add a scalable modal so that zoom in and out.
> 
> <img alt="image" width="630" src="https://user-images.githubusercontent.com/2016594/160071067-e34c395b-4f27-4ab1-b002-fb0d73b517b8.png">
> 

Thanks for the review! I like the idea of adding a resizer to the popover. However I'd suggest implementing it as a separate PR as there are a few edge cases to consider